lib/http.js

```diff
@@ -81,17 +81,55 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     cors(),
     // FIXME: add validation
     //validate({bodySchema: updateMeterBody}),
-    _createAuthorizeMiddleware(),
+    _addMeterRecordToRequest(),
+    _createAuthorizeMiddleware({
+      getExpectedTarget({req}) {
+        // only support update meter config URL as expected target; controller
+        // is the controller of the meter
+        const baseUrl = `https://${req.get('host')}${routes.basePath}`;
+        const expectedTarget =
+          `${baseUrl}/${encodeURIComponent(req.params.meterId)}`;
+        return {expectedTarget};
+      },
+      getRootController({req}) {
+        // root controller for update meter config is always the controller of
+        // the meter
+        const {meter: {controller}} = req.meterRecord;
+        return controller;
+      }
+    }),
     asyncHandler(async (req, res) => {
       let {body: meter} = req;
 
       // update meter according to handler result
       ({meter} = await HANDLERS.update({meter}));
 
-      // FIXME: update meter using meters.update()
+      const {id: meterId, controller, sequence} = meter;
```

Comment:
We should run this check before `HANDLERS.update` runs because it seems to be a user input problem that can be handled without having to call the handler.